spec/models/api/hses_spec.rb

```diff
@@ -21,28 +21,86 @@
   end
 
   describe "Issues" do
-    let(:issues) { Api::Hses::Issues.new(user: user) }
+    context "with params argument" do
+      let(:issues) {
+        Api::Hses::Issues.new(
+          user: user,
+          params: ActionController::Parameters.new({page: "2"})
```

Comment:
i don't think i've used `ActionController::Parameters` directly before. is there an advantage to using that, or just passing in the hash directly?

Reply:
I was just trying to be as realistic as possible, since presumably if we're passing parameters in from the controller that's the type we would be getting. The test will be fine with just a hash if you have a preference for that.